app/src/main/res/values-v21/styles.xml

```diff
@@ -1,15 +1,30 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
-
-    <style name="calculatorButton">
+    <style name="CalculatorButton">
```

Comment:
as you create CalculatorOperationsButtons, here maybe tha name could be CalculatorOperatorButtons

Reply:
Done